### PR TITLE
test(e2e): Exam Components/finish-exam-modal

### PR DESCRIPTION
**Pull request by dzmdre**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

 [#51705](https://github.com/freeCodeCamp/freeCodeCamp/issues/51705)

### CODE REVIEW

**Comment by dzmdre** [comment]
camperbot cant code

**Comment by dzmdre** [comment]
Looks like I have no "pnpm run seed:exams" on ci and can not click on the Start exam button
https://discord.com/channels/692816967895220344/715074489422970962/1159520223645409321

**Comment by dzmdre** [comment]
Blocked by #51841

**Comment by dzmdre** [comment]
Please review

**Comment by dzmdre** [comment]
@naomi-lgbt please take a look

**Review by naomi-lgbt — APPROVED** [review]

File: client/src/templates/Challenges/exam/show.tsx [hunk 1]
```diff
@@ -415,33 +415,39 @@ class ShowExam extends Component<ShowExamProps, ShowExamState> {
                   <Spacer size='large' />
                   <div className='exam-answers'>
                     {generatedExamQuestions[currentQuestionIndex].answers.map(
-                      ({ answer, id }) => (
-                        <label className='exam-answer-label' key={id}>
+                      (item, index) => (
+                        <label
+                          className='exam-answer-label'
+                          data-playwright-test-label={
+                            'exam-answer-label-' + String(index)
```
[the remark this answers is not in the document]
Comment (dzmdre) [reply client/src/templates/Challenges/exam/show.tsx]: @huyenltnguyen Done

**Review by huyenltnguyen — APPROVED** [review]
LGTM 👍 Thanks @dzmdre!

**Pull request merged**